zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/intent/FeelExpressionResolutionIntent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol.record.intent;
+
+public enum FeelExpressionResolutionIntent implements Intent {
+  RESOLVE(0),
+  RESOLVED(1);
```

Comment:
🔧 For the intent itself, I'd use:

```
EVALUATE(0),
EVALUATED(1)
```

in order to match the `DecisionEvaluationIntent`